src/app/services/getAllServiceUsers.js

```diff
@@ -50,7 +50,11 @@ const getServiceUsers = async (req, res) => {
       return;
     }
 
-    const usersOfService = await servicesStorage.getAllUsersOfService(serviceId, pageNumber, pageSize, req.header('x-correlation-id'));
+    let userIds;
+    if (req.body && req.body.userIds)
```

Comment:
this whole function is where we have to handle that the call can be a GET or a POST, define the params in a way that we can get them for both scenarios:
  const paramsSource = req.method === 'POST' ? req.body : req.query;
then use paramsSource in all the places in the function